File: Selenium with Java/New Project/src/First/First.java
package First;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class First {
	WebDriver driver;
	
	//launching firefox
	public void LaunchBrowser() throws InterruptedException {
		System.setProperty("webdriver.gecko.driver","D:/Projects/Installables/geckodriver.exe");
		driver = new FirefoxDriver();
		driver.navigate().to("https://www.simplilearn.com");
	}
	//searching for selenium course and clicking on it
	public void search() throws InterruptedException {
		driver.findElement(By.id("header_srch")).sendKeys("selenium");
		driver.findElement(By.xpath("/html/body/div[2]/div/div[1]/header/div/div/div/nav/div[3]/div[1]/form/button[3]/span")).click();
		Thread.sleep(3000);
		driver.findElement(By.xpath("//h4[text()='Automation Testing Masters Program']")).click();
		Thread.sleep(3000);
		System.out.println("The Page Title is- "+driver.getTitle());
	
	}
	//CLosing the Browser
	public void close() {
		driver.quit();
	}
	
	public static void main(String[] args) throws InterruptedException {
		First obj= new First();
		obj.LaunchBrowser();
		obj.search();
		obj.close();

	}

}

File: Selenium with Python/first.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

driver=webdriver.Firefox()

driver.get("https://opensource-demo.orangehrmlive.com/")

wait = WebDriverWait(driver, 10)
username = wait.until(EC.presence_of_element_located((By.NAME, "username")))
password = wait.until(EC.presence_of_element_located((By.NAME, "password")))

driver.find_element(By.NAME,"username").send_keys("Admin")

driver.find_element(By.NAME,"password").send_keys("admin123")

driver.find_element(By.XPATH,"/html/body/div/div[1]/div/div[1]/div/div[2]/div[2]/form/div[3]/button").click()

actual_title=driver.title
expected_title="OrangeHRM"

if actual_title == expected_title:
    print("Title matches")
else:
    print(f"Actual title is {actual_title}")

driver.close()